experimental/addons/cronhpa/template.cue

```diff
@@ -0,0 +1,37 @@
+package main
+
+output: {
+	apiVersion: "core.oam.dev/v1beta1"
+	kind:       "Application"
+	spec: {
+		components: [ {
+			type: "k8s-objects"
+			name: "cronhpa-rbac"
+			properties: objects: [
+				controllerClusterRole,
+				controllerRoleBinding,
+			]
+		},
+			{
+				type: "k8s-objects"
+				name: "cronhpa-CRD"
+				properties: objects: [
+					cronhpaCRD,
+				]
+			},
+			cronController]
+		policies: [{
+			type: "topology"
```

Comment:
We can add a keep resource GC policy for CRD resource. Thus, disabling this  addon wouldn't affect the application. Please refer to : https://github.com/kubevela/catalog/blob/374ce88aca03e7963a14bbd060a83547ae9330fc/addons/fluxcd/template.cue#L90

Reply:
> We can add a keep resource GC policy for CRD resource. Thus, disabling this addon wouldn't affect the application. Please refer to :
> 
> https://github.com/kubevela/catalog/blob/374ce88aca03e7963a14bbd060a83547ae9330fc/addons/fluxcd/template.cue#L90

updated